### ISSUE TITLE
winget package

### ISSUE THREAD

**Issue by flriancu** [wxFormBuilder/wxFormBuilder#800]
Hi, it would be nice to have a winget package, so one can run e.g. `winget install wxformbuilder` on Windows 10 and later.

**Comment by sodevel** [comment]
Of course it would be nice to have wxFB available on whatever package manager there is, but there are just too many and i simply don't have the ressources to support them.

**Comment by lebao3105** [comment]
![Screenshot_20240725-000808.png](https://github.com/user-attachments/assets/b4f4a4d4-2ae1-4442-8863-a0499b706308)

![Screenshot_20240725-000735.png](https://github.com/user-attachments/assets/647448a4-5209-4ebe-ae11-c371173c4a60)

I tried to publish one, and got these noticeable things in the agreements

**Comment by lebao3105** [comment]
Policies about packages to be submitted: https://learn.microsoft.com/en-us/windows/package-manager/package/windows-package-manager-policies

**Comment by sodevel** [comment]
I am not sure what is the question here? You are trying to add wxFB to winget and have to sign a CLA for this?

I did only a quick look at how winget works, but it looks like in constrast to Flathub, third parties can add packages because all what winget stores is basically the installer? Since wxFB is GPL software and you didn't modify it at all, there should be no legal issue to add it to winget. However, i don't know how that CLA works and what boxes you need to tick there.

**Comment by lebao3105** [comment]
> I am not sure what is the question here? You are trying to add wxFB to winget and have to sign a CLA for this?
> 
> I did only a quick look at how winget works, but it looks like in constrast to Flathub, third parties can add packages because all what winget stores is basically the installer? Since wxFB is GPL software and you didn't modify it at all, there should be no legal issue to add it to winget. However, i don't know how that CLA works and what boxes you need to tick there.

Yes, I have to sign the CLA when I make a PR there. The policies allow wxFB to have things like wxWidgets and plugins (as far as I understand the policy)

> 1.2.3
> The Product may contain fully integrated middleware (such as third-party cross-platform engines and third-party analytics services).

> The Product may depend on non-integrated software (such as another Product, module, or service) to deliver its primary functionality.

Same as you, I don't quite understand CLA so I captured parts of the CLA and sent them here. Expect someone to help.

This is the first time I actually care and discover about these license stuff.
